extensions/amp-smartlinks/0.1/linkmate-options.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {userAssert} from '../../../src/log';
+
+/**
+ * Get the config values from the tag on the amp page
+ * @param {!Element} element
+ * @return {!Object}
+ */
+export function getConfigOptions(element) {
+  return {
+    nrtvSlug: getNrtvAccountName_(element),
+    linkmateEnabled: getLinkmateFlag_(element),
+    exclusiveLinks: getExclusiveLinksFlag_(element),
+    linkAttribute: getLinkAttribute_(element),
+    linkSelector: getLinkSelector_(element),
+  };
+}
+
+/**
+ * The slug used to distinguish Narrativ accounts.
+ * @param {!Element} element
+ * @return {string}
+ * @private
+ */
+function getNrtvAccountName_(element) {
+  const nrtvSlug = element.getAttribute('nrtv-account-name');
+  userAssert(nrtvSlug,
```

Comment:
there is a validator rule that would catch this so the userAssert shouldn't be required.

Reply:
addressing in a separate PR?

Reply:
Whoops completely misread this comment initially. I'll update the PR with to remove this assert.

Reply:
updated